src/https/utilities/https-utilities

```diff
@@ -26,6 +26,8 @@ generate_self_signed_certificate()
 	openssl req -newkey rsa:4096 -nodes -keyout $SELF_SIGNED_KEY \
 	        -x509 -days 90 -out $SELF_SIGNED_CERT -subj "/O=Nextcloud"
 
+	openssl ecparam -name secp384r1 >> $SELF_SIGNED_CERT
```

Comment:
[Doesn't sound like this is necessary](https://weakdh.org/sysadmin.html) as of Apache v2.4.8. `SSLOpenSSLConfCmd DHParameters` is enough. Can you verify?